frontend/src/metabase/querying/components/FilterModal/utils/filters.ts

```diff
@@ -15,8 +15,8 @@ export function appendStageIfAggregated(query: Lib.Query) {
     : query;
 }
 
-export function dropStageIfEmpty(query: Lib.Query) {
-  return Lib.dropStageIfEmpty(query, -1);
+export function dropEmptyStages(query: Lib.Query) {
```

Comment:
We should inline this function as it no longer serves any purpose

Reply:
done